libbeat/cmd/instance/metrics.go

```diff
@@ -122,16 +122,16 @@ func reportBeatCPU(_ monitoring.Mode, V monitoring.Visitor) {
 
 	monitoring.ReportNamespace(V, "user", func() {
 		monitoring.ReportInt(V, "ticks", int64(cpuTicks.User))
-		monitoring.ReportInt(V, "time", userTime)
+		monitoring.ReportInt(V, "time_ms", userTime)
```

Comment:
Can we go with `time.ms` to follow our metricbeat convention?

Reply:
Would that allow the field name of the monitoring document to keep the `time_ms` name?

Reply:
@tsullivan I assume the metrics in the monitoring document are in sync with what we have here so it's `time`?

Reply:
@ruflin the concern with naming the field `time` is that it's ambiguous on whether this is a measure of milliseconds or a unix epoch value. That's our x-pack monitoring data mappings don't have any fields named `time` and we want to stay consistent with that practice.

Whether it is `time_ms` or `time.ms` is fine with me. My concern with `time.ms` is that the dot should represent a path in the json, not a dot in the field name.